src/lib/addMatchSummary.tsx

```diff
@@ -0,0 +1,274 @@
+        losers +
+        " so hard, they respawned in Ohio &#128128 &#128293; dropping a diff of  " +
+        scoreDiff +
+        " points like it's nothing"
+      );
+    case scoreDiff > 140:
+      return (
+        "deleted " +
+        losers +
+        " off the map with " +
+        scoreDiff +
+        " points - literally a war crime"
+      );
+    case scoreDiff > 120:
+      return (
+        "humiliated " +
+        losers +
+        " on main &#127909;&#128514; — cooked up a " +
+        scoreDiff +
+        " point diff for real for real"
+      );
+    case scoreDiff > 110:
+      return (
+        "absolutely pulled up with a main character energy " +
+        losers +
+        " got side charactered with " +
+        scoreDiff +
+        " points, no cap"
+      );
+    case scoreDiff > 100:
+      return (
+        "caught " +
+        losers +
+        " lacking in 4K &#128249, that's a war crime with " +
+        scoreDiff +
+        " points"
+      );
+    case scoreDiff > 90:
+      return (
+        "absolutely demolished " +
+        losers +
+        " &#128165; - took it home with " +
+        scoreDiff +
+        " points, no cap"
+      );
+    case scoreDiff > 80:
+      return (
+        "gaslit, girlbossed & gatekeeped &#128131;&#128170; " +
+        losers +
+        " by " +
+        scoreDiff +
+        " points no cap"
+      );
+    case scoreDiff > 60:
+      return (
+        "made " +
+        losers +
+        " question their life choices - walked off with " +
+        scoreDiff +
+        " points like a menace &#128520;"
+      );
+    case scoreDiff > 50:
+      return (
+        "lowkey cooked " +
+        losers +
+        " for dinner - took it by " +
+        scoreDiff +
+        " points, no filter"
+      );
+    case scoreDiff > 45:
+      return (
+        "had all the skibidi rizz against " +
+        losers +
+        " and flexed a " +
+        scoreDiff +
+        " point win, no cap"
+      );
+    case scoreDiff > 40:
+      return (
+        "had " +
+        losers +
+        " fighting demons &#128128;&#128553; — still lost by " +
+        scoreDiff +
+        " points LMAO"
+      );
+    case scoreDiff > 35:
+      return (
+        "gaslit " +
+        losers +
+        " out of existence - cleaned " +
+        scoreDiff +
+        " points ahead &#128128;"
+      );
+    case scoreDiff > 30:
+      return (
+        "outplayed " +
+        losers +
+        "  while scrolling TikTok - secured a " +
+        scoreDiff +
+        " diff effortlessly"
+      );
+    case scoreDiff >= 25:
+      return (
+        " went alpha-lite mode on " +
+        losers +
+        " and dropped a " +
+        scoreDiff +
+        " diff like it's light work"
+      );
+    case scoreDiff >= 20:
+      return (
+        "dropped a mild menace win." +
+        losers +
+        " still crying, by losing with " +
+        scoreDiff +
+        " point diff"
+      );
+    case scoreDiff >= 10:
```

Comment:
This should be `>= 5` (The minimum a game can be won)
https://leaderboard-preview.faust.space/result/937
Otherwise we end up with these.
<img width="309" height="32" alt="Image" src="https://github.com/user-attachments/assets/84872e0e-2f22-4cf2-8a20-4586ad7e1288" />